File: div_2_A/38/A.cpp
#include <iostream>
#include <string>
#include <bits/stdc++.h>
#include <set>
using namespace std;

int main()
{


    set<char, greater<char> > s1;
 
    // insert elements in random order
    s1.insert("40");
    s1.insert("30");
    s1.insert("60");
    s1.insert("20");
    s1.insert("50");
     
    // only one 50 will be added to the set
    s1.insert("50");
    s1.insert("10");
 
    // printing set s1
    set<char, greater<char> >::iterator itr;
    cout << "\nThe set s1 is : \n";
    for (itr = s1.begin(); itr != s1.end(); itr++)
    {
        cout << *itr<<" ";
    }
    // int t, n, d;
    // // cin >> t;
    // cin >> n;
    // string s;
    // long long a, ans = 0;
    // cin >> s;
    // for(int i = 0; i < n; ++i){
    //     if()
    // }

    // cout << ans << endl;
}

File: div_2_A/38/A.py
n = int(input())
s = input()

v = {'a','e','i','o','u','y'}

ans = ''
ans+=s[0]
for i in range(1,n):
    if s[i] in v and s[i-1] in v:
        continue
    else:
        ans+=s[i]
# if s[-1] in v and s[-2] in v :
#     print(ans)
# else:
#     ans+=s[-1]
#     print(ans)
print(ans)